### ISSUE TITLE
Allow a way to modify slugs

### ISSUE THREAD

**Issue by alexjoverm** [nuxt/content#15]
### Is your feature request related to a problem? Please describe.
Having a routing convention is great! However, in medium-large blogs or content-based websites, you usually want a way to create your url slugs independently from your file structure.

For instance, you may want urls like:
- /:slug
- /:year/:month/:slug
- ...

Right now, the only way to achieve this is to create a file structure accordingly. But for these cases it could be problematic.

Imagine this case: you have a blog with 200 articles and you want urls like `/:slug`. That'd mean to have everything under `content` dir, which is crazy in orgs terms. (yes, you could have dirs like `content/:year`, but that's not the url you want for SEO purposes whatsoever)

### Describe the solution you'd like
In short: a way to **decouple** file structure and urls shape

I know is not the same case, but as inspiration Gridsome have a [Pages API](https://gridsome.org/docs/dynamic-routing/#programmatic-dynamic-routes) useful for this case.

**Comment by Atinux** [comment]
Hey @alexjoverm 

Actually $content() has to be used in your Nuxt pages/ to query the content

You can always have pages/_year/_month.vue that sill query another directory inside content/.

Would that solve the issue?

**Comment by alexjoverm** [comment]
Oh, my bad. Yes it does indeed. There was something I misunderstood, so it's actually perfect the way it is :)

**Comment by maemaemae3** [comment]
I have a question related to this issue.
Can't I place foobar.md file in `/content/[year]/[month]/foobar.md` and make the article url like `http://mysite.com/foobar` ?
(instead of `http://mysite.com/[year]/[month]/foobar`)

Isn't there any options like putting `permalink: foo` in [Front Matter](https://content.nuxtjs.org/writing/#front-matter) so that I can specify the url of each article?
